avalon/tools/loader/model.py

```diff
@@ -109,6 +111,15 @@ def __init__(
         self.asset_doc_projection = asset_doc_projection
         self.subset_doc_projection = subset_doc_projection
 
+        self.sync_server_settings = None
+        proj_settings = get_project_settings(dbcon.Session["AVALON_PROJECT"])
+        if proj_settings.get('global').get('sync_server')['enabled']:
```

Comment:
If you don't specify default return value of get method use rather `proj_settings['global']['sync_server']['enabled']`. Result is the same but shorted.